src/components/survey/Survey.tsx

```diff
@@ -0,0 +1,90 @@
+"use client"
+import { ChangeEvent, useState, useCallback } from 'react';
+
+
+interface SurveyProps {
+  title: string;
+  lowest: string;
+  highest: string;
+}
+
+export const Survey = ({ title, lowest, highest  }: SurveyProps) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [selectedRating, setSelectedRating] = useState("0");
+
+	const handleRatingClick = (event: any) => {
+		setSelectedRating(event.target.value);
+	};
+
+  const handleSubmit = (event: any) => {
+    event.preventDefault();
+    // alert(`Rating: ${selectedRating}`);
+    console.log(`Submitted rating: ${selectedRating}`);
+  };
+
+
+  return (
+    <div className="absolute bottom-0 right-0">
```

Comment:
If we want to refactor this component as to my previous comment, I think it makes sense to abstract this to it's parent.